docs/build/sdks/js/zksync-ethers/providers.md

```diff
@@ -382,7 +384,7 @@ import { Provider, types } from "zksync-ethers";
 
 const provider = Provider.getDefaultProvider(types.Network.Sepolia);
 const account = "0x36615Cf349d7F6344891B1e7CA7C72883F5dc049";
-const tokenAddress = "0x765F5AF819D818a8e8ee6ff63D8d0e8056DBE150"; // Crown token which can be minted for free
```

Comment:
I thought `0x765F5AF819D818a8e8ee6ff63D8d0e8056DBE150` is the correct address for CROWN in Sepolia?

Reply:
The old token did not work well with paymaster, so the old token and paymaster is switched with new ones.

Reply:
What did not work? Is this something we need to fix in other places of the docs?

Reply:
The deployed paymaster contract is not verified so I could not check the reason. The error that is throw is `Invalid magic value`, so something with signature is not correct. I updated to all places in docs including other SDKs.